docs/develop-load-manager.md

```diff
@@ -207,3 +103,216 @@ then all machines are approximately overloaded. In the case in which a broker's
+
+  ```bash
+  ===================================================================================================================
+  ||COUNT          |TOPIC          |BUNDLE         |PRODUCER       |CONSUMER       |BUNDLE +       |BUNDLE -       ||
+  ||               |4              |4              |0              |2              |0              |0              ||
+  ||RAW SYSTEM     |CPU %          |MEMORY %       |DIRECT %       |BW IN %        |BW OUT %       |MAX %          ||
+  ||               |0.25           |47.94          |0.01           |0.00           |0.00           |47.94          ||
+  ||ALLOC SYSTEM   |CPU %          |MEMORY %       |DIRECT %       |BW IN %        |BW OUT %       |MAX %          ||
+  ||               |0.20           |1.89           |               |1.27           |3.21           |3.21           ||
+  ||RAW MSG        |MSG/S IN       |MSG/S OUT      |TOTAL          |KB/S IN        |KB/S OUT       |TOTAL          ||
+  ||               |0.00           |0.00           |0.00           |0.01           |0.01           |0.01           ||
+  ||ALLOC MSG      |MSG/S IN       |MSG/S OUT      |TOTAL          |KB/S IN        |KB/S OUT       |TOTAL          ||
+  ||               |54.84          |134.48         |189.31         |126.54         |320.96         |447.50         ||
+  ===================================================================================================================
+  ```
+
+  - Modular load manager
+
+  ```bash
+  ===================================================================================================================
+  ||SYSTEM         |CPU %          |MEMORY %       |DIRECT %       |BW IN %        |BW OUT %       |MAX %          ||
+  ||               |0.00           |48.33          |0.01           |0.00           |0.00           |48.33          ||
+  ||COUNT          |TOPIC          |BUNDLE         |PRODUCER       |CONSUMER       |BUNDLE +       |BUNDLE -       ||
+  ||               |4              |4              |0              |2              |4              |0              ||
+  ||LATEST         |MSG/S IN       |MSG/S OUT      |TOTAL          |KB/S IN        |KB/S OUT       |TOTAL          ||
+  ||               |0.00           |0.00           |0.00           |0.00           |0.00           |0.00           ||
+  ||SHORT          |MSG/S IN       |MSG/S OUT      |TOTAL          |KB/S IN        |KB/S OUT       |TOTAL          ||
+  ||               |0.00           |0.00           |0.00           |0.00           |0.00           |0.00           ||
+  ||LONG           |MSG/S IN       |MSG/S OUT      |TOTAL          |KB/S IN        |KB/S OUT       |TOTAL          ||
+  ||               |0.00           |0.00           |0.00           |0.00           |0.00           |0.00           ||
+  ===================================================================================================================
+  ```
+
+ - Extensible load manager
+   
+   ```bash
+   tbd
```

Comment:
@Demogorgon314 as we discussed just now, can you provide technical input for this?

Reply:
BTW, the extensible load manager should use the following command:
```
bin/pulsar-perf monitor-brokers --connect-string pulsar://localhost:6650 --extensions
```